docs/agents/spec-process.md

```diff
@@ -0,0 +1,69 @@
+Every change that affects more than one agent should be initiated via a change to the specification rather than creating a cross-agent issue.
```

Comment:
```suggestion
Every change that affects more than one agent should be initiated via a change to the specification.  Once accepted, cross-agent issues can follow.
```

Reply:
I updated the suggestion, I meant per-agent, not cross-agent

Reply:
As this is described in more detail in the last step, I wouldn't want to anticipate it here.
If you find the term `a cross-agent issue` ambiguous, we change it to this:

```suggestion
Every change that affects more than one agent should be initiated via a PR against the specification rather than creating an issue that describes the changes in its description.
```